tests/functional/test_custom_task_templates.py

```diff
@@ -84,13 +92,6 @@ def project_multiclass_skyserver(self):
                 "pytorch_drop_in_env",
                 "binary",
             ),
-            (
```

Comment:
Note: removing tests below that no longer are applicable. Before we had model-metadata.yaml working completely you could take the same code folder and apply target types. Usually this worked because there was logic in the fit hook to switch between classification and regression & there was no score hook (so DataRobot dynamically applied the right scoring functions). Now with model-metadata.yaml we have to specify the type, so this approach won't work.

After discussing with Jesse we decided not to create a bunch of new pipeline templates (because they're being discouraged in the future and test time considerations). Instead I just removed all the tests that won't work